src/theory/bags/inference_manager.cpp

```diff
@@ -0,0 +1,37 @@
+/*********************                                                        */
+/*! \file inference_manager.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds, Paul Meng
```

Comment:
Update to your name.  Similar for other places

Reply:
Done